gcp/modules/gcp-stackdriver-monitoring/main.tf

```diff
@@ -23,6 +28,7 @@ resource "template_dir" "resources" {
     project_id                = "${var.project_id}"
     domain_name               = "${var.domain_name}"
     ssl_enabled_uptime_checks = "${var.ssl_enabled_uptime_checks}"
+    notification_email        = "${var.env == "dev" ? var.auth_user_email : var.notification_email}"
```

Comment:
How about changing this to `notification_email = "${var.notification_email != ""} ? var.notification_email : var.auth_user_email"`?

Reply:
This looks much better, since we don't have to rely on env. Thanks!